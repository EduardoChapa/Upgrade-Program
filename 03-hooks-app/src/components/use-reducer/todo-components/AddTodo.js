import React from 'react';

function AddTodo({ handleSubmit, handleChange, description }) {
    return (
        <div className='col-5'>
            <h4>Add todo</h4>
            <hr/>
            <form onSubmit={ handleSubmit }>
                <input 
                    type='text'
                    name='description'
                    placeholder='Description...'
                    autoComplete='off'
                    onChange={ handleChange }
                    value={ description }
                />
                <button className='btn btn-outline-primary mt-1 btn-block'>Add</button>
            </form>
        </div> 
    );
}

export default AddTodo;