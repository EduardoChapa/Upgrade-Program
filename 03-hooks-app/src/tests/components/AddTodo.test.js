import React from 'react';
import { shallow } from 'enzyme';
import AddTodo from '../../components/use-reducer/todo-components/AddTodo';

describe('Testing AddTodo component', () => {
    const wrapper = shallow(<AddTodo />);

    test('should display correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});