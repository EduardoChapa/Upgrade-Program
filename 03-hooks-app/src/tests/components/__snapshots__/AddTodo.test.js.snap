// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing AddTodo component should display correctly 1`] = `
<div
  className="col-5"
>
  <h4>
    Add todo
  </h4>
  <hr />
  <form>
    <input
      autoComplete="off"
      name="description"
      placeholder="Description..."
      type="text"
    />
    <button
      className="btn btn-outline-primary mt-1 btn-block"
    >
      Add
    </button>
  </form>
</div>
`;
